src/services/flows/service/flows.ts

```diff
@@ -181,3 +181,40 @@ export const create = function (options?, sdkOptions?) {
  * An alias for `create` to match the API documentation.
  */
 export const deploy = create;
+
+/**
+ * Update a Flow
+ * @see https://globusonline.github.io/globus-flows/#tag/Flows/paths/~1flows~1{flow_id}/put
+ *
+ * This is documented as a PUT request in the API documentation, but it allows partial documents
+ * as the payload, so it follows the PATCH semantics.
```

Comment:
```suggestion
```

I'm not sure this comment is necessary, or maybe I'm just misunderstanding. It seems like it should be the responsibility of the service to describe the behavior of their resources rather than us doing it here. Said another way, the `@see` to the documentation seems like it would be enough?

Reply:
I think you're probably right.